rdt/transformers/categorical.py

```diff
@@ -197,7 +197,10 @@ def _reverse_transform(self, data):
                 labels.append(key)
 
         result = pd.cut(data, bins=bins, labels=labels, include_lowest=True)
-        result = result.replace(nan_name, np.nan)
+        current_categories = result.cat.categories
+        if nan_name in current_categories:
+            result = result.cat.remove_categories(nan_name)
```

Comment:
why do you need the temp variable `current_categories`? You can just directly do 
```python
if nan_name in result.cat.categories
```